tensorflow/lite/micro/examples/mnist_lstm/evaluate_test.py

```diff
@@ -68,15 +70,15 @@ def testCompareWithTFLite(self):
       self.assertAllLess((tflite_output - tflm_output), 1e-5)
 
   def testInputErrHandling(self):
-    wrong_size_image_path = self.sample_image_dir + 'resized9.png'
+    wrong_size_image_path = os.path.join(PREFIX_PATH, "samples",
+                                         'resized9.png')
```

Comment:
super nit: let's make this
```py
os.path.join(PREFIX_PATH, "samples/resized9.png")
```

that way it is easier to see the path relative to the python file.

Reply:
Fixed.